client/src/modules/stock/exit/exit.js

```diff
@@ -29,21 +29,29 @@ function StockExitController(
   vm.gridApi = {};
   vm.selectedLots = [];
   vm.selectableInventories = [];
+  vm.currentInventories = [];
   vm.reset = reset;
   vm.ROW_ERROR_FLAG = bhConstants.grid.ROW_ERROR_FLAG;
+  vm.overconsumption = [];
 
   vm.onDateChange = date => {
     vm.movement.date = date;
     if (vm.movement.date < new Date()) {
       vm.dateMessageWarning = true;
     }
     loadInventories(vm.depot, date);
+    loadCurrentInventories(vm.depot);
+
+    vm.overconsumption = [];
     checkValidity();
   };
 
   vm.onChangeDepot = depot => {
     vm.depot = depot;
     loadInventories(vm.depot);
+    loadCurrentInventories(vm.depot);
+
```

Comment:
How does this handle the case where the current inventories may already have negative values?

Reply:
For products with negative quantities, the system will assimilate to products out of stock, which is why only those with positive quantities are filtered.
````
 function loadCurrentInventories(depot, dateTo = new Date()) {
    vm.loading = true;
    Stock.lots.read(null, { depot_uuid : depot.uuid, dateTo })
      .then(lots => {
        vm.currentInventories = lots.filter(item => item.quantity > 0);
      })
      .catch(Notify.handleError)
      .finally(() => {
        vm.loading = false;
      });
  }
````

Reply:
ok!

Reply:
ok. I understand!